pkg/client/core/coretester.go

```diff
@@ -0,0 +1,34 @@
+package core
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/juju/errors"
+
+	"github.com/bitnami-labs/charts-syncer/api"
+	"github.com/bitnami-labs/charts-syncer/pkg/client/chartmuseum"
+)
+
+// ClientV2Tester defines the methods that a fake tester should implement
+type ClientV2Tester interface {
```

Comment:
shouldn't this file be called `clienttester`, then?

Reply:
just renamed